src/test/java/com/dabsquared/gitlabjenkins/util/CommitStatusUpdaterTest.java

```diff
@@ -116,7 +117,7 @@ public void setUp() throws Exception {
 	public void test() {
 		CommitStatusUpdater.updateCommitStatus(build, taskListener, BuildState.success, STAGE);
 		
-		verify(client).changeBuildStatus(Integer.toString(PROJECT_ID), REVISION, BuildState.success, null, STAGE, Jenkins.getInstance().getRootUrl() + BUILD_URL, null);
+		verify(client).changeBuildStatus(Integer.toString(PROJECT_ID), REVISION, BuildState.success, null, STAGE, Jenkins.getInstance().getRootUrl() + BUILD_URL + BUILD_URL_REDIRECT, null);
```

Comment:
I think using DisplayURLProvider.get().getRunURL(build) as in GitLabCommitStatusPublisherTest would be even better...

Reply:
That is correct! I fixed it.